### PR TITLE
EDU-1350 Show document inputs on room page

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1350

### CODE REVIEW

File: src/components/pages/room.yml [hunk 1]
```diff
@@ -11,3 +11,6 @@ room:
   cancelRoomMembership:
     en: Cancel room membership
     de: Raum-Mitgliedschaft beenden
+  documentInputs:
+    en: Document inputs
+    de: Dokumenteingabe
```
Comment (ahelmberger) [inline src/components/pages/room.yml:16]: Maybe let's also put the plural here in German "Dokumenteingaben"?

File: src/components/room/room-document-inputs.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+roomDocumentInputs:
+  emptyStateTitle:
+    en: No document inputs have been submitted in this room
```
Comment (ahelmberger) [inline src/components/room/room-document-inputs.yml:3]: And maybe here singular in English?

Comment (adrianaluchian) [reply src/components/room/room-document-inputs.yml]: Indeed, streamline towards singular.

File: src/components/pages/room.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pages/room.yml]: Done, thanks!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**